packages/gatsby-theme-newrelic/gatsby-node.js

```diff
@@ -156,6 +158,13 @@ exports.sourceNodes = (
     tessen: tessen
       ? { product: tessen.product, subproduct: tessen.subproduct }
       : null,
+    googleTagManager: googleTagManager
+      ? {
+          trackingId: googleTagManager.trackingId,
+          src: googleTagManager.src,
+          options: googleTagManager.options,
+        }
+      : null,
```

Comment:
I _think_ we could simplify this to:
```suggestion
    googleTagManager: googleTagManager || null,
```